doi-transformer/src/main/java/no/unit/nva/doi/transformer/CrossRefConverter.java

```diff
@@ -347,6 +354,7 @@ private boolean hasEditor(CrossRefDocument document) {
     private BookAnthology createBookAnthology(CrossRefDocument document) {
         return new BookAnthology.Builder()
                 .withPeerReviewed(hasReviews(document)) // Same as in BasicContext
+                .withPages(extractOrDummyMonograpPages(document))
```

Comment:
This method is very specific and in essence overrides the existing method extractPages.

Method extractPages should have signature:

`private Pages extractPages(CrossRefDocument)` and should return the correct pages for the document type under consideration.